api/docs/tough-cookie.getpublicsuffixoptions.md

```diff
@@ -0,0 +1,16 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [tough-cookie](./tough-cookie.md) &gt; [GetPublicSuffixOptions](./tough-cookie.getpublicsuffixoptions.md)
+
+## GetPublicSuffixOptions type
+
+Options for configuring how [getPublicSuffix()](./tough-cookie.getpublicsuffix.md) behaves.
+
+**Signature:**
+
+```typescript
+export type GetPublicSuffixOptions = {
+    allowSpecialUseDomain?: boolean | undefined;
+    ignoreError?: boolean | undefined;

```

Comment:
Can we get the properties annotated, as well?

Reply:
yeah, i also noticed that individual properties didn't show up in the docs after the fact.  but i may have forgot to regenerate these pages after annotating the type.

Reply:
@wjhsf turns out that the doc tooling won't populate the property information when using a `type` vs. an `interface` so I've switched `export type GetPublicSuffixOptions` to `export interface GetPublicSuffixOptions`.